powerlibs/django/restless/contrib/endpoints/base.py

```diff
@@ -86,3 +86,33 @@ class SoftDeletableListEndpointMixin:
     def get_query_set(self, request, *args, **kwargs):
         queryset = super().get_query_set(request, *args, **kwargs)
         return queryset.filter(deleted=False)
+
+
+class SieveListEndpoint:
+
+    def post(self, request, *args, **kwargs):
+
+        for field_name in ('deleted', 'updated_at', 'created_at'):
+            if field_name in request.data:
+                return Http400({"error": "Invalid field name: {}".format(field_name)})
+
+        return super().post(request, *args, **kwargs)
+
+
+class SieveDetailEndpoint:
+
+    def put(self, request, *args, **kwargs):
+
+        for field_name in ('deleted', 'updated_at', 'created_at'):
+            if field_name in request.data:
+                return Http400({"error": "Invalid field name: {}".format(field_name)})
+
+        return super().put(request, *args, **kwargs)
+
+    def patch(self, request, *args, **kwargs):
+
+        for field_name in ('deleted', 'created_by', 'updated_at', 'created_at'):
```

Comment:
Why is there 4 fields here and only 3 on line 106?

Reply:
Coloquei tudo junto. 

``` Não permitir que atualize-se (via put ou patch) campos como `created_by`, `created_at` e `updated_at`.```

e 

```SoftDeletableEndpointMixin, sumir com o campo deleted no GET e impedir que seja setado via POST/PUT/PATCH.```